QuickLayout/QuickLayout.Touch/AppDelegate.cs

```diff
@@ -1,8 +1,8 @@
-using MonoTouch.Foundation;
-using MonoTouch.UIKit;
-using Cirrious.CrossCore;
-using Cirrious.MvvmCross.Touch.Platform;
-using Cirrious.MvvmCross.ViewModels;
+using Foundation;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.iOS.Platform;
+using MvvmCross.Platform;
+using UIKit;
 
 namespace QuickLayout.Touch
```

Comment:
sub-project folder name, *.csproj file name, and namespaces are renamed. 
de609d20fd827dea89e0ed4ebcc77eb288488d36
739267bc2d6bc29a54a8eace37bc26247d34b3b8
de609d20fd827dea89e0ed4ebcc77eb288488d36